### PR TITLE
Fix PHPUnit Tests

### PR DESCRIPTION
**Pull request by admdly**
Fix error with PHPUnit tests introduced by changes to `FOSSBilling\Validate` class.

### CODE REVIEW

File: src/modules/Support/Service.php [hunk 1]
```diff
@@ -798,8 +798,6 @@ public function ticketCreateForAdmin(\Model_Client $client, \Model_SupportHelpde
 
     public function ticketCreateForGuest($data)
     {
-        $this->di['validator']->isEmailValid($data['email']);
```
Comment (BelleNottelling) [inline src/modules/Support/Service.php:801]: We really *have* to do input validation here. Both `ticketCreateForGuest` and `publicTicketCreate` are used to create tickets for non-clients, so the email param is coming from an un-trusted source.

**Review by BelleNottelling — CHANGES_REQUESTED** [review]

File: src/modules/Support/Service.php (hunk 1 above)
Comment (admdly) [reply src/modules/Support/Service.php:801]: Agree - but `isEmailValid` no longer exists. Perhaps reworking around `FOSSBilling\Tools` `validateAndSanitizeEmail()`?

Comment (BelleNottelling) [reply src/modules/Support/Service.php:801]: Yeah, that'll work great to perform the validation / sanitation needed here

Comment (admdly) [reply src/modules/Support/Service.php:801]: Updated to use `Tools` email validation function, including the related tests. 🙂

**Review by evrifaessa — APPROVED** [review]
LGTM.

**Review by BelleNottelling — APPROVED** [review]
Thanks for fixing that :)

**Pull request merged**